draft-ietf-mls-protocol.md

```diff
@@ -4662,14 +4657,9 @@ in an asynchronous application, it may be the case that all members that could
 detect a fault in a Commit are offline.  In such a case, the Commit will be
 accepted by the group, and the resulting state possibly used as the basis for
 further Commits.  When the affected members come back online, they will reject
-the first commit, and thus be unable to catch up with the group.
-
-Applications can address this risk by requiring certain members of the group to
-acknowledge successful processing of a Commit before the group regards the
-Commit as accepted.  The minimum set of acknowledgements necessary to verify
-that a Commit is well-formed comprises an acknowledgement from one member per
-node in the UpdatePath, that is, one member from each subtree rooted in the
-copath node corresponding to the node in the UpdatePath.
```

Comment:
Why remove this? It sounds like a viable strategy in some cases.

Reply:
I thought about this more and I still don't like this section:
- It says you need one acknowledgement per node in the UpdatePath, but that's not really true. You would need one acknowledgement per *ciphertext* in the UpdatePath since the ciphertexts are independent.
- That's assuming that everybody votes honestly. If the attacker colludes with someone to vote dishonestly, then the same attack still works. So I need acknowledgement from *everybody* in the group.

Reply:
There's really not a reasonable way to prevent this issue. Can only provide reporting mechanism and make it right when it happens